cmds/rsdp/rsdp.go

```diff
@@ -60,7 +60,7 @@ func getRSDP(path string) (string, error) {
 		case res := <-channel:
 			if strings.Contains(res, "RSDP") {
 				returnValue = strings.Split(res, " ")[2]
-				exit = 0
+				exit = 1
```

Comment:
var exit bool

default value will be false
for dataRead == 1 && ! exit
exit = true

The loop needs a bit more work but this minimal change will make this PR ok